pkg/pvstore/pvstore.go

```diff
@@ -101,6 +112,34 @@ func RunCreate(cmd *cobra.Command, args []string) {
 	util.Panic(err)
 }
 
+// RunDelete runs a CLI command
+func RunDelete(cmd *cobra.Command, args []string) {
+	log := util.Logger()
+
+	if len(args) != 1 || args[0] == "" {
+		log.Fatalf(`❌ Missing expected arguments: <pv-store-name> %s`, cmd.UsageString())
+	}
+
+	nbClient := system.GetNBClient()
+
+	poolName := args[0]
+
+	err := nbClient.DeletePoolAPI(nb.DeletePoolParams{
+		Name: poolName,
+	})
+	if err != nil {
```

Comment:
What if pv-pool doesn't exist?

Reply:
then it will panic... we prefer something else? I guess it is pretty bad if to happens

Reply:
Probably best to just continue if the pool doesn't exist, because it means the deletion is essentially successful

Reply:
added handling for doesn't exist - so all these cases will have same kind of responses on failures